components/brave_rewards/resources/rewards_panel/lib/initial_state.ts

```diff
@@ -10,7 +11,10 @@ export function getInitialState (): HostState {
     loading: true,
     requestedView: null,
     rewardsEnabled: false,
-    balance: 0,
+    balance: {
```

Comment:
Can we make this an `Optional<number>` and perform the "is loading" logic in `extension_api_adapter`?